docs/README.md

```diff
@@ -3081,12 +2887,201 @@ console.log(sql.toXML());
 
 ## iUserSpace
 
-The `iUserSpace` class is deprecated. Use the [Toolkit](#Class-Toolkit) class instead.
```

Comment:
Why was the deprecation notice removed?

Reply:
Its now listed under deprecated classes section and Toolkit user space functions no longer exist.

Reply:
I think we should clarify what that means, though. We should change the existing wording to say that it will be removed in version 2

Reply:
That makes sense. The other `Remove * functions from Toolkit` PRs that got merged have the same problem.

Can we fix these all up in a follow up PR?

Reply:
Sure, sounds good.